src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a class="logo" href="#home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <div class="header-menu">
+        <nav class="nav">
+          <ul class="list">
+            <li class="nav__list">
+              <a class="nav__link is-active" href="#">Apple</a>
+            </li>
+            <li class="nav__list">
+              <a class="nav__link" href="#"> Samsung </a>
+            </li>
+            <li class="nav__list">
+              <a class="nav__link" href="#">Smartphones</a>
+            </li>
+            <li class="nav__list">
+              <a
+              class="nav__link"
+              href="#"
+              data-qa="hover"
+              >
```

Comment:
```suggestion
              <a
                class="nav__link"
                href="#"
                data-qa="hover"
              >
```
Consider fixing indents according to the style guide kit